modules/nf-core/ngscheckmate/patterngenerator/tests/main.nf.test

```diff
@@ -16,7 +16,10 @@ nextflow_process {
             script "../../../bowtie/build/main.nf"
             process {
                 """
-                input[0] = file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
+                input[0] = [
+                    [ id: 'sarscov2' ],
+                    file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
```

Comment:
Has there been a formal switch to the `params.modules_testdata_base_path + hard_path` syntax now?

Reply:
Yes... although I'm also assuming at some point there is a bulk update.